armi/reactor/geometry.py

```diff
@@ -131,48 +443,17 @@ def __str__(self):
 VALID_GEOMETRY_TYPE = {HEX, HEX_CORNERS_UP, RZT, RZ, CARTESIAN}
 
 FULL_CORE = "full"
-THIRD_CORE = "third "
-QUARTER_CORE = "quarter "
-EIGHTH_CORE = "eighth "
-SIXTEENTH_CORE = "sixteenth "
+THIRD_CORE = "third"
+QUARTER_CORE = "quarter"
+EIGHTH_CORE = "eighth"
+SIXTEENTH_CORE = "sixteenth"
 REFLECTIVE = "reflective"
 PERIODIC = "periodic"
+NO_SYMMETRY = "no symmetry"
 THROUGH_CENTER_ASSEMBLY = (
-    " through center assembly"  # through center assembly applies only to cartesian
+    "through center assembly"  # through center assembly applies only to cartesian
```

Comment:
We probably want to make "assembly" optional. No need to address this now, though